slides/k8s/setup-managed.md

```diff
@@ -0,0 +1,253 @@
+# Installing a managed cluster
+
+*"The easiest way to install Kubernetes is to get someone
+else to do it for you."
+<br/>
+([Jérôme Petazzoni](https://twitter.com/jpetazzo))*
+
+- Let's see a few options to install managed clusters!
+
+- This is not an exhaustive list
+
+  (the goal is to show the actual steps to get started)
+
+- All the options mentioned here require an account
+with a cloud provider
+
+- ... And a credit card
+
+---
+
+## EKS (the hard way)
+
+- [Read the doc](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+- Create service roles, VPCs, and a bunch of other oddities
+
+- Try to figure out why it doesn't work
+
+- Start over, following an [official AWS blog post](https://aws.amazon.com/blogs/aws/amazon-eks-now-generally-available/)
+
+- Try to find the missing Cloud Formation template
+
+--
+
+.footnote[(╯°□°)╯︵ ┻━┻]
```

Comment:
lololol

Reply:
:D

Reply:
(Number of hours spent by me, trying to automate provisioning of EKS clusters before learning about eks-cli: way too much!)